src/openfermion/measurements/__init__.py

```diff
@@ -17,3 +17,8 @@
 
 from ._rdm_equality_constraints import (one_body_fermion_constraints,
                                         two_body_fermion_constraints)
+
+from ._rdm_mapping_functions import (krond, tpdm_to_tqdm, tpdm_to_opdm,
```

Comment:
I think some of these names might be a bit terse, `opdm` and `tpdm` are clear enough IMO but I don't know what `oqdm`, `tqdm`, or `phdm` are. This might be clearer to someone who works with RDMs more, so maybe this isn't needed - @babbush / @jarrodmcc ?

Reply:
I agree with Ian that these names are too terse.